Google.PowerShell/BigQuery/GcbqDataset.cs

```diff
@@ -136,6 +136,68 @@ protected override void ProcessRecord()
 
     /// <summary>
     /// <para type="synopsis">
+    /// Updates information in an existing dataset.
+    /// </para>
+    /// <para type="description">
+    /// Updates information in an existing dataset. If no Project is specified, the default project will be used. 
+    /// This cmdlet returns a Dataset object.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> $updatedSet | Set-GcbqDataset</code>
+    ///   <para>This will update the values stored in the cloud for the dataset passed via pipeline.</para>
```

Comment:
I think the comment here could be more explicit.

This command will update the values stored in the BigQuery dataset for the default project passed.  

I would also add more context as to what " $updatedSet " is.